rust-runtime/aws-smithy-http-server-python/src/tls/listener.rs

```diff
@@ -188,10 +188,11 @@ mod tests {
             // We have a certificate with invalid date, so request should fail
             let config = client_config_with_cert(&invalid_cert);
             let response = make_req(config, &addr).await;
+            println!("{}", response.as_ref().unwrap_err().to_string());
```

Comment:
Maybe we can pass `response.as_ref().unwrap_err().to_string()` as second argument to below `assert!` statement

Reply:
This is a leftover when I was trying to fix the tests for the new version of `rustls`. I'll remove this line.